packages/Kokkos/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -39,10 +41,8 @@ spatial_query( BVH<NO> const bvh, Predicate const &predicate, int *indices,
 #if HAVE_DTK_DBC
             if ( n_indices > max_n_indices )
                 printf( "Increase the size of indices array\n" );
-            assert( n_indices > max_n_indices );
-#else
-            (void)max_n_indices;
 #endif
+            assert( n_indices < max_n_indices );
```

Comment:
You need these two lines to avoid the unused parameter warning and to close the preprocessor if statement.